README.md

```diff
@@ -50,6 +50,10 @@ On error, it will return an error object like this:
 }
 ```
 
+### Notes
+
+- On NodeJS, you can set the env variable `FETCH_KEEP_ALIVE` to `true` to enable keep-alive connections.
```

Comment:
we have to set this ENV on all projects again? 🙈 maybe we can set it true from code itself using dotenv?? 

https://github.com/motdotla/dotenv#processenv

Reply:
We can set it inside .env yes

Reply:
No i mean without having to change env on digital ocean

Reply:
Yes, by adding it to each project .env file. If they’re not using dotenv, still need to change do env

Reply:
Cool, will change in this PR itself?

Reply:
No, the env var has to be added to the service using this package